WISE-Community/WISE-API
# feat(Peer Group): Add teacher endpoint to get PeerGroup info for activity

### Changes
- Added new teacher endpoint ```/api/teacher/peer-group-info/RUN_ID/NODE_ID/COMPONENT_ID``` that will return PeerGroups and Workgroups that aren't in PeerGroups for the specified activity
- Requires database change: add periodId bigint field to peer_groups table

### Test Prep
- add periodId column to peer_groups table: ```alter table peer_groups add column periodId bigint;```

### Test
- Log in as a teacher
- Make request to ```/api/teacher/peer-group-info/RUN_ID/NODE_ID/COMPONENT_ID```
   - If valid runId/nodeId/componentId combination and teacher does not have permissions, should see AccessDenied
   - If invalid runId/nodeId/componentId combination, should throw error
   - If valid runId/nodeId/componentId combination and teacher has permissions, should return a map containing two objects:
      - "peerGroups": list of PeerGroups for the activity in the run
      - "workgroupsNotInPeerGroup": list of Workgroups that are not in a PeerGroup for the activity

Closes #57
